lib/response.js

```diff
@@ -266,7 +266,7 @@ module.exports = {
   redirect (url, alt) {
     // location
     if (url === 'back') url = this.ctx.get('Referrer') || alt || '/'
-    if (url.startsWith('https://') || url.startsWith('http://')) {
+    if (/^https?:\/\//i.test(url)) {
```

Comment:
@FDrag0n I change to use Regex to avoid lowerCase twice.

Reply:
It's a nice change👍